Ambibox with menu and new auto switching for video

KenV99 opened the pull request:
I fixed missing info in addon.xml and settings.xml

KenV99 committed:
New Menu and Display Aspect Ratio Switching

KenV99 committed:
trapped uninitialized videomode and set default

KenV99 committed:
Trap errors in media.py

needs byte compiled version, pyo file deleted for now

KenV99 committed:
small fixes

KenV99 committed:
Added Google translate in Russian of new strings

KenV99 committed:
Update default.py

AmbiBox commented:
Re: [AmbiBox-XBMC] Ambibox with menu and new auto switching for video (#1)
Where disappear add-on (service.py) in XBMC (in addon.xml)?http://bsvi.ru/putpic/upload/2014_01_17/4226a7.jpg 
You can merge this Pull Request by running  git pull https://github.com/KenV99/AmbiBox-XBMC master
Or view, comment on, or merge it at:
  https://github.com/AmbiBox/AmbiBox-XBMC/pull/1Commit Summary
New Menu and Display Aspect Ratio Switching
trapped uninitialized videomode and set default
Trap errors in media.py
small fixes
Added Google translate in Russian of new strings
File Changes
M script.ambibox/addon.xml (4) 
M script.ambibox/changelog.txt (8) 
M script.ambibox/default.py (262) 
A script.ambibox/resources/data/dardata.xml (28) 
M script.ambibox/resources/language/English/strings.xml (17) 
M script.ambibox/resources/language/Russian/strings.xml (21) 
M script.ambibox/resources/lib/Media.py (14) 
M script.ambibox/resources/settings.xml (20) 
Patch Links:
https://github.com/AmbiBox/AmbiBox-XBMC/pull/1.patch
https://github.com/AmbiBox/AmbiBox-XBMC/pull/1.diff
—

## Reply to this email directly or view it on GitHub.

С уважением,
 AmbiBox                          mailto:AmbiBox@mail.ru

KenV99 committed:
Fixed addon,xml

I fixed missing reference in addon,xml and put the option to turn off
menu directly back into settings.xml. I am unsure that is needed with
the logic introduced with the "video_choice" menu option. With that,
there are four choices: 1) use a default profile, 2) switch based on
aspect ratio, 3) show a menu, or 4) turn off the leds.
